tests/playwright/sanity/video.test.js

```diff
@@ -44,6 +48,85 @@ test.describe( 'Video tests inside a container', () => {
 		// Verify that the container has an equal height to the video iFrame.
 		expect( containerHeight ).toEqual( videoIframeHeight );
 	} );
+
+	const videos = require( _path.resolve( __dirname, `../testData/video.json` ) );
+
+	for ( const video in videos ) {
+		test( `${ video } controls and link test`, async ( { page }, testInfo ) => {
+			const player = videos[ video ];
+			const wpAdmin = new WpAdminPage( page, testInfo );
+			const editor = new EditorPage( page, testInfo );
+			const videoWidget = new VideoWidget( page, testInfo );
+			const startTime = '10';
+			const endTime = '20';
+
+			await wpAdmin.openNewPage();
+			await editor.closeNavigatorIfOpen();
+			await editor.addWidget( 'video' );
+			await videoWidget.selectVideoSource( video );
+
+			await videoWidget.setTime( 'Start', startTime );
+			if ( 'youtube' === video ) {
+				await videoWidget.setTime( 'End', endTime );
+				await videoWidget.selectSuggestedVideos( 'Any Video' );
+			}
+
+			const controls = await new Promise( ( resolved ) => {
+				resolved( player.controls.map( ( control ) => {
+					return EditorSelectors.video[ control ];
+				} ) );
+			} );
```

Comment:
I don't think we need a `Promise` here:
```suggestion
			const controls = player.controls.map( ( control ) => {
				return EditorSelectors.video[ control ];
			} );
```

Reply:
fixed